core/server/fluxruntime.go

```diff
@@ -168,7 +166,7 @@ func (cs *coreServer) GetChildObjects(ctx context.Context, msg *pb.GetChildObjec
 		Kind:    msg.GroupVersionKind.Kind,
 	})
 
-	if err := clustersClient.List(ctx, msg.ClusterName, &l, client.InNamespace(msg.Namespace)); err != nil {
+	if err := clustersClient.List(ctx, msg.ClusterName, &l); err != nil {
```

Comment:
@yitsushi This would only work if the user has access to list all namespaces, right?. I think this is missing the cross-namespace query logic.

Reply:
NVM I see that `wego-system` doesn't have permissions to list namespace but this still works as you describe. I must be missing something.